src/algorithms.jl

```diff
@@ -31,7 +32,13 @@ ExplicitRK(;tableau=ODE_DEFAULT_TABLEAU) = ExplicitRK(tableau)
 @inline trivial_limiter!(u, f, t) = nothing
 
 struct Euler <: OrdinaryDiffEqAlgorithm end
-struct RichardsonEuler <: OrdinaryDiffEqAlgorithm end
+struct RichardsonEuler <: OrdinaryDiffEqExtrapolationVarOrderVarStepAlgorithm
+  max_order::Int
+  min_order::Int
+  init_order::Int
+end
+RichardsonEuler(;max_order=9,min_order=1,init_order=9) = RichardsonEuler(max_order,min_order,init_order)
```

Comment:
init at order 9 might be high. I'd put it to 5

Reply:
In the initialising phase, work per step is checked for k = 1 to max_order. And the order was going as high as 7 in some cases. That's why I had put it as 9. If there can be other problems due to high init_order, I'll decrease this one.

Reply:
The problem you were testing on was completely perfect for high order. Not all problems are like that.

Reply:
okay.. will set init_order as 5 and leave max_order as 9. Will it be fine?

Reply:
yes

Reply:
done